docs/howto/deploy-subgraph/deploying-subgraph.md

```diff
@@ -0,0 +1,995 @@
+
+## 1 What is The graph
+
+For an NFT or gaming application, we can find directly the owner of a NFT or an item based on their ID.  
+For doing this, we only have to request a read operation on the contract.  
+But more advanced queries like finding all NFT or items associated to a user is not possible only by querying the read method's on contract.  
+For doing this we would have to process every transfer event emmitted, agregate data and process.
+This is not possible for a dapp only running on a browser.  
+With a server thinks are not much better, because this require a centralized cache to store data and thus it introduce a security weakness.  
+That is why The Graph network was created.
+
+The graph is an opensource software based on a decentralized protocol that work as a cache  
+for indexing and querying data from blockchain.  
+It was initially created for ethereum blockchain and then started to integrate other blockchain. 
+Were as a classic database, on blockchains you cannot perform a query on data by criteria.
+
+The Graph retrieves blocks and extract data before storing it into indexed tables, called Subgraphs, allowing any application  
+to perform a query to its protocol and receive the response.  
+
+The Graph comes with its own crypto currency called GRT.
+
+its protocol include 3 types of actors:
+
+Curators: They are subgraph developers who assess which subgraphs need to be indexed by The Graph.  
+They needs to attach GRT to these subgraphs.    
+
+Indexers: Node operators tasked with providing indexing and querying services for the signaled subgraphs,  
+and must stake GRT in order to provide these services.  
+
+Delegators: Delegate GRT to indexers in order to contribute to running the network without installing a node.  
+
+## 2 Install the Graph 
```

Comment:
Don't add manually the numerotation to the titles

Reply:
Ok